openpype/pipeline/colorspace.py

```diff
@@ -20,10 +23,55 @@
 
 
 class CachedData:
-    remapping = {}
-    allowed_exts = {
```

Comment:
⚠️ The removal of this attribute has broken publishing since it's still used on [line 940 of this file](https://github.com/ynput/OpenPype/blob/5d1f2b0d9ed3ff44f1f924870e9a829c9ee4ee12/openpype/pipeline/colorspace.py#L940). ⚠️ 

@jakubjezek001